Jellyfin.Server/Middleware/BaseUrlRedirectionMiddleware.cs

```diff
@@ -65,8 +65,9 @@ public async Task Invoke(HttpContext httpContext, IServerConfigurationManager se
                 // Always redirect back to the default path if the base prefix is invalid or missing
                 _logger.LogDebug("Normalizing an URL at {LocalPath}", localPath);
 
-                var uri = new Uri(localPath);
-                var redirectUri = new Uri(baseUrlPrefix + "/" + _configuration[DefaultRedirectKey]);
+                var port = httpContext.Request.Host.Port ?? (httpContext.Request.Scheme == "https" ? 443 : 80);
```

Comment:
Is it necessary?

Reply:
I think port can not be set if it's the default port. It does not need to be set per definition though. I can check it tomorrow if you want to prevent it.

Reply:
I see no reason to set it when we don't need to. Defaulting to -1 should leave it out of the uri